### ISSUE TITLE
Mip-Nerf-360 Data Converting

### ISSUE THREAD

**Issue by j93hahn** [nerfstudio-project/nerfstudio#1438]
Hi, has anyone been able to convert the Mip-NeRF-360 datasets used here https://jonbarron.info/mipnerf360/ into the specified NeRFStudio format, via the ns-process-data command? I seem unable to get the COLMAP operations to work and am wondering if anyone has had luck with that.

**Comment by tancik** [comment]
Thats odd, it worked for us. Is it all of the datasets, or just one of them?

**Comment by 97littleleaf11** [comment]
It seems that the camera type of official datasets (generated by Colmap) hasn't been supported by nerfstudio. Using `ns-download-data` to get the supported data from nerfstudio team might work. Check this branch https://github.com/nerfstudio-project/nerfstudio/pull/1139 for more details.

**Comment by jkulhanek** [comment]
However, if you use the original images from the dataset, they are downscaled in a different way. After downscaling the width and height are rounded up, which is different from nerfstudio, where it is assumed they are rounded down. You can use ns-process-data to downsample the images or manually modify `cameras/cameras.py:799` to round up instead.

**Comment by j93hahn** [comment]
> It seems that the camera type of official datasets (generated by Colmap) hasn't been supported by nerfstudio. Using `ns-download-data` to get the supported data from nerfstudio team might work. Check this branch #1139 for more details.

Hi thank you for your response; which dataset is it listed under? When I run `ns-download-data -h`, I do not see the MipNerf360 dataset as an option.

**Comment by j93hahn** [comment]
> Thats odd, it worked for us. Is it all of the datasets, or just one of them?

Yeah I am unable to get any of them to work -- I am running into many errors with the CUDA backend/binary file inconsistencies and cannot even get COLMAP to properly work. Would it be possible to share the datasets? I would like to experiment with all of them but am particularly interested in `bonsai`, `garden`, and `stump`.

**Comment by j93hahn** [comment]
Okay thanks @tancik @97littleleaf11 @jkulhanek. I am using the datasets taken directly off of MipNeRF360 and am just going to convert the `poses_bounds.npy` files into the `transforms.json` files. Is there a script you ran to run this? All the datasets have already been processed by COLMAP so I don't think I need that

**Comment by jkulhanek** [comment]
Can you try the `--skip-colmap` option in `ns-process-data` images?

**Comment by jkulhanek** [comment]
I guess you can also add `--skip-process-images`

**Comment by yyzzyy78** [comment]
Excuse me, have you solved this problem? @j93hahn @jkulhanek 

I tried to generate transforms.json with: ns-process-data images --data /360_v2/garden/images/ --output-dir /360_v2/garden/studio_test --num-downscales 4  --skip-colmap --colmap-model-path /360_v2/garden/sparse/0/

And I want to train and test nerfacto with /images_4 with resolution 1926,840, but the generated transforms.json has resolution 5187*3361. So I manually set self.downscale_factor=4 in [https://github.com/nerfstudio-project/nerfstudio/blob/main/nerfstudio/data/dataparsers/nerfstudio_dataparser.py#L276](url).

After that, I trained the model with: ns-train nerfacto --data /360_v2/garden/studio_test/ --output-dir /360_v2/garden/studio_test/output --steps-per-eval-batch 500 --steps-per-eval-image 500 --steps-per-eval-all-images 5000 --max-num-iterations 60001 --experiment-name garden --pipeline.model.predict-normals False --vis viewer+tensorboard

The psnr result I got is 20.997 with 60000 iterations. Have I did something wrong?
I am looking forward to your reply!

**Comment by yyzzyy78** [comment]
Besides, I have tested the instant-ngp model with the above operation, and I got pnsr=24.08 with 60000 iterations and psnr=24.30 with 250000 iterations.

**Comment by yyzzyy78** [comment]
Sorry, the above link is:
https://github.com/nerfstudio-project/nerfstudio/blob/f2fdd10d9bca5d66286e72fcd87c41122050156f/nerfstudio/data/dataparsers/nerfstudio_dataparser.py#L276

**Comment by jkulhanek** [comment]
Sorry, I was away from a PC for a while. Can you please try with camera pose optimisation disabled?
